### PR TITLE
Feature/admin reports

### PR DESCRIPTION
**Pull request by YourSouLi5Mine**
Acceptance criteria:
- When server starts a job is created
- The job sends mails every minute with the admin_report.pdf

### CODE REVIEW

File: app/workers/admin_report_worker.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+class AdminReportWorker
+  include Sidekiq::Worker
+
+  def perform(*args)
```
Comment (ricvillagrana) [inline app/workers/admin_report_worker.rb]: Unused variable `args`

Comment (YourSouLi5Mine) [reply app/workers/admin_report_worker.rb]: :eyes: deleting it

File: app/mailers/weekly_mailer.rb [hunk 2]
```diff
@@ -0,0 +1,14 @@
+class WeeklyMailer < ApplicationMailer
+  default from: "precichysim@gmail.com"
+
+  def weekly
```
Comment (OmauryMarmolejo) [inline app/mailers/weekly_mailer.rb]: Not sure if you need to name the method `weekly` if you're naming the class `WeeklyMailer`

Comment (YourSouLi5Mine) [reply app/mailers/weekly_mailer.rb]: :eyes: solved :D

**Pull request merged**